### PR TITLE
[13.0][MIG] base_m2m_custom_field

### PR DESCRIPTION
**Pull request by grindtildeath**
Forward-port of #1738

### CODE REVIEW

File: base_m2m_custom_field/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Camptocamp SA
```
Comment (Tonow-c2c) [inline base_m2m_custom_field/__manifest__.py:1]:
```suggestion
# Copyright 2020 Camptocamp SA
```

File: base_m2m_custom_field/fields.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Camptocamp SA
```
Comment (Tonow-c2c) [inline base_m2m_custom_field/fields.py:1]:
```suggestion
# Copyright 2020 Camptocamp SA
```

**Review by Tonow-c2c — APPROVED** [review]

**Comment by grindtildeath** [comment]
@Tonow-c2c No need to update the date if the code was written last year :wink:

**Review by simahawk — APPROVED** [review]

**Pull request merged**